MapboxNavigation/RoutePageViewController.swift

```diff
@@ -56,13 +56,13 @@ class RoutePageViewController: UIPageViewController {
 extension RoutePageViewController: UIPageViewControllerDataSource, UIPageViewControllerDelegate {
     func pageViewController(_ pageViewController: UIPageViewController, viewControllerAfter viewController: UIViewController) -> UIViewController? {
         let controller = viewController as! RouteManeuverViewController
-        let stepAfter = maneuverDelegate.stepAfter(controller.step)
+        let stepAfter = maneuverDelegate.stepAfter(controller.step!)
```

Comment:
@1ec5 Is there a reason for changing the step property to an optional, but force unwrapping it here?